tests/integration/integration_test_Entity.py

```diff
@@ -141,7 +141,7 @@ def test_Entity():
 
     newfolder = Folder('Testing Folder', parent=project)
     newfolder = syn.store(newfolder)
-    link = Link(newfolder.id, parent = folder.id)
```

Comment:
I meant should you be testing both (passing in ID or an entity object)? I'm fine with not just wondering what we do in other tests that create Synapse objects.